frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -299,6 +305,10 @@ export const experimentLogic = kea<experimentLogicType>({
             if (values.experimentId === 'new') {
```

Comment:
sounds like this should be?

```suggestion
            if (values.experimentId === 'new' || values.editingExistingExperiment) {
```

Reply:
Ah, I see, this destroys whatever insight you have if you change the trend/funnel selector while editing. Makes sense.